README.md

```diff
@@ -42,8 +42,7 @@ $ cmake ..
 $ cmake --build .
 ```
 
-This will build Fizzy as a library and since there is no public API
-(the so called *embedder API* in WebAssembly) yet, this is not very useful.
+This will build Fizzy as a library. [C API] is provided for embedding Fizzy engine in applications, as well as [Rust binding].
```

Comment:
I think for Rust we should give an actual example using the published crate, and perhaps link to `crates.io` and/or `docs.rs` (for documentation).

- https://crates.io/crates/fizzy
- https://docs.rs/fizzy